lib/salus/report.rb

```diff
@@ -187,9 +136,44 @@ def export_report
 
     private
 
-    # Wrap lines to 100 chars by default
-    def wrap(text, line_width = 100)
-      text.gsub(/(.{1,#{line_width}})/, "\\1\n")
+    def deep_stringify_keys(datum)
+      case datum
+      when Hash then datum.map { |key, value| [key.to_s, deep_stringify_keys(value)] }.to_h
+      when Array then datum.map { |value| deep_stringify_keys(value) }
+      else datum
+      end
+    end
+
+    # FIXME(as3richa): factor this out along with the identical version
+    # in scan_report.rb. Maybe have a Formatting mixin module for wrapping,
```

Comment:
recommend the mixim module option

Reply:
I see that the methods are slightly different so fine to leave this as a future TODO like you have.